test/core/image_processing/simple_kernels.cpp

```diff
@@ -0,0 +1,142 @@
+#include <boost/gil/image_processing/numeric.hpp>
+#include <boost/gil/image.hpp>
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/typedefs.hpp>
+#include <boost/core/lightweight_test.hpp>
+
+namespace gil = boost::gil;
+
+bool are_equal(gil::gray32f_view_t expected, gil::gray32f_view_t actual)
```

Comment:
I know this is not grammatically sound, but should read `is_equal`  like in `is_same<T1, T2>` :)

Reply:
It turns out the function is not even used, so I removed it